feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -302,6 +302,7 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
     { customerId }: { customerId?: string },
     parameters?: AsmDeepLinkParameters
   ): void {
+    this.showCreateCustomerSuccessfullyAlert = false;
```

Comment:
avoid having `showCreateCustomerSuccessfullyAlert` not closed for the previous created customer